web/src/lightbox.js

```diff
@@ -318,6 +318,7 @@ export function build_open_media_function(on_close) {
     if (on_close === undefined) {
         on_close = function () {
             $(".player-container iframe").remove();
+            remove_video_player();
```

Comment:
I moved this call to the two places where we already remove the Youtube/etc. players in declaring `on_close` handlers, since that seemed easier to maintain.